packages/palette/src/elements/LineChart/Line.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { color } from "../../helpers"
+
+/**
+ * Line is the component responsible for rendering a line that goes through all of the data points
+ * in a line chart.
+ */
+export const Line = ({ d }: { d: string }) => {
+  return <path fill="none" stroke={color("black10")} d={d} />
```

Comment:
Not sure if `d` follows prior art but 1-character var names don't convey much information, at least passed as props.